### PR TITLE
Fix errors in the documentation

### PR DESCRIPTION
**Pull request by PaulWessel**
This was a mix of typos, missing modifiers, wrong modifiers, and lack of clarity.

### CODE REVIEW

File: doc/rst/source/events.rst [hunk 1]
```diff
@@ -18,7 +18,7 @@ Synopsis
 [ |-C|\ *cpt* ]
 [ |-D|\ [**j**\|\ **J**]\ *dx*\ [/*dy*][**+v**\ [*pen*]] ]
 [ |-E|\ **s**\|\ **t**\ [**+o**\|\ **O**\ *dt*][**+r**\ *dt*][**+p**\ *dt*][**+d**\ *dt*][**+f**\ *dt*][**+l**\ *dt*] ]
-[ |-F|\ [**+a**\ *angle*][**+f**\ *font*][**+j**\ *justify*][**+r**\ [*first*]\|\ **+z**\ [*format*]] ]
+[ |-F|\ [**+a**\ *angle*][**+f**\ *font*][**+j**\ *justify*][**+r**\ [*first*]\|\ **z**\ [*format*]] ]
```
Comment (seisman) [inline doc/rst/source/events.rst]: This one looks very confusing.

File: doc/rst/source/explain_fft.rst_ [hunk 2]
```diff
@@ -1,4 +1,4 @@
-**-N**\ [**a**\|\ **f**\|\ **m**\|\ **r**\|\ **s**\|\ *nx/ny*][**+a**\|\ [**+d**\|\ **h**\|\ **l**][**+e**\|\ **n**\|\ **m**][**+t**\ *width*][**+v**][**+w**\ [*suffix*]][**+z**\ [**p**]]
+**-N**\ [**a**\|\ **f**\|\ **m**\|\ **r**\|\ **s**\|\ *nx/ny*][**+a**\ \|\ **d**\ \|\ **h**\ \|\ **l**][**+e**\|\ **n**\|\ **m**][**+t**\ *width*][**+v**][**+w**\ [*suffix*]][**+z**\ [**p**]]
```
Comment (seisman) [inline doc/rst/source/explain_fft.rst_]:
```suggestion
**-N**\ [**a**\|\ **f**\|\ **m**\|\ **r**\|\ **s**\|\ *nx/ny*][**+a**\|\ **d**\|\ **h**\|\ **l**][**+e**\|\ **n**\|\ **m**][**+t**\ *width*][**+v**][**+w**\ [*suffix*]][**+z**\ [**p**]]
```

File: doc/rst/source/gmtmath.rst [hunk 3]
```diff
@@ -17,7 +17,7 @@ Synopsis
 [ |-E|\ *eigen* ] [ |-I| ]
 [ |-N|\ *n\_col*\ [/*t_col*] ]
 [ |-Q| ] [ |-S|\ [**f**\|\ **l**] ]
-[ |-T|\ [*min*/*max*/*inc*\ [**+n**]\|\ *file*\|\ *list*] ]
+[ |-T|\ [*min*/*max*/*inc*\ [**+b**\ \|\ **l**\ \|\ **n**]\|\ *file*\|\ *list*] ]
```
Comment (seisman) [inline doc/rst/source/gmtmath.rst]:
```suggestion
[ |-T|\ [*min*/*max*/*inc*\ [**+b**\|\ **l**\|\ **n**]\|\ *file*\|\ *list*] ]
```

File: doc/rst/source/gmtmath.rst [hunk 4]
```diff
@@ -138,7 +138,7 @@ Optional Arguments
 
 .. _-T:
 
-**-T**\ [*min*/*max*/*inc*\ [**+n**]\|\ *file*\|\ *list*]
+**-T**\ [*min*/*max*/*inc*\ [**+b**\ \|\ **l**\ \|\ **n**]\|\ *file*\|\ *list*]
```
Comment (seisman) [inline doc/rst/source/gmtmath.rst]:
```suggestion
**-T**\ [*min*/*max*/*inc*\ [**+b**\|\ **l**\|\ **n**]\|\ *file*\|\ *list*]
```

File: doc/rst/source/grdmask.rst [hunk 5]
```diff
@@ -17,7 +17,7 @@ Synopsis
 |SYN_OPT-R|
 [ |-A|\ [**m**\|\ **p**\|\ **x**\|\ **y**] ]
 [ |-N|\ [**z**\|\ **Z**\|\ **p**\|\ **P**]\ *values* ]
-[ |-S|\ *search\_radius* \|\ *xlim*\ /*ylim* ] [ |SYN_OPT-V| ]
+[ |-S|\ *search\_radius*\ \|\ *xlim*\ /*ylim* ] [ |SYN_OPT-V| ]
```
Comment (seisman) [inline doc/rst/source/grdmask.rst]:
```suggestion
[ |-S|\ *search\_radius*\|\ *xlim*\ /*ylim* ] [ |SYN_OPT-V| ]
```

File: doc/rst/source/grdmask.rst [hunk 6]
```diff
@@ -104,7 +104,7 @@ Optional Arguments
 
 .. _-S:
 
-**-S**\ *search\_radius* \|\ *xlim*\ /*ylim*
+**-S**\ *search\_radius*\ \|\ *xlim*\ /*ylim*
```
Comment (seisman) [inline doc/rst/source/grdmask.rst]:
```suggestion
**-S**\ *search\_radius*\|\ *xlim*\ /*ylim*
```

File: doc/rst/source/makecpt.rst [hunk 7]
```diff
@@ -24,7 +24,7 @@ Synopsis
 [ |-N| ]
 [ |-Q| ]
 [ |-S|\ *mode* ]
-[ |-T|\ [*min*/*max*/*inc*\ [**+n**]\|\ *file*\|\ *list*] ]
+[ |-T|\ [*min*/*max*/*inc*\ [**+b**\ \|\ **l**\ \|\ **n**]\|\ *file*\|\ *list*] ]
```
Comment (seisman) [inline doc/rst/source/makecpt.rst]:
```suggestion
[ |-T|\ [*min*/*max*/*inc*\ [**+b**\|\ **l**\|\ **n**]\|\ *file*\|\ *list*] ]
```

File: doc/rst/source/psevents.rst [hunk 8]
```diff
@@ -18,7 +18,7 @@ Synopsis
 [ |-C|\ *cpt* ]
 [ |-D|\ [**j**\|\ **J**]\ *dx*\ [/*dy*][**+v**\ [*pen*]] ]
 [ |-E|\ **s**\|\ **t**\ [**+o**\|\ **O**\ *dt*][**+r**\ *dt*][**+p**\ *dt*][**+d**\ *dt*][**+f**\ *dt*][**+l**\ *dt*] ]
-[ |-F|\ [**+a**\ *angle*][**+f**\ *font*][**+j**\ *justify*][**+r**\ [*first*]\|\ **+z**\ [*format*]] ]
+[ |-F|\ [**+a**\ *angle*][**+f**\ *font*][**+j**\ *justify*][**+r**\ [*first*]\|\ **z**\ [*format*]] ]
```
Comment (seisman) [inline doc/rst/source/psevents.rst]: This one is confusing, too.

File: doc/rst/source/supplements/geodesy/earthtide.rst [hunk 9]
```diff
@@ -68,11 +68,11 @@ Optional Arguments
 
 .. _-C:
 
-**-C**\ *x|e,y|n,z|v*
+**-C**\ **x**\ \|\ **e**\ ,\ **y**\ \|\ **n**\ ,\ **z**\ \|\ **v**
```
Comment (seisman) [inline doc/rst/source/supplements/geodesy/earthtide.rst]:
```suggestion
**-C**\ **x**\|\ **e**,\ **y**\|\ **n**,\ **z**\|\ **v**
```

File: doc/rst/source/supplements/geodesy/earthtide.rst [hunk 10]
```diff
@@ -68,11 +68,11 @@ Optional Arguments
...
     Select which component to write to individual grids. Requires **-G**.
     Append comma-separated codes for available components: **x** or **e** for the east component;
     **y** or **n** for the north component; and **z** or **v** for the vertical component.
-    For example, **-C**\ *e,v*, will write 2 grids. One with east and other with the vertical components.
+    For example, **-Ce**\ ,\ **v**, will write 2 grids. One with east and other with the vertical components.
```
Comment (seisman) [inline doc/rst/source/supplements/geodesy/earthtide.rst]:
```suggestion
    For example, **-Ce**,\ **v**, will write 2 grids. One with east and other with the vertical components.
```

File: doc/rst/source/psevents.rst (hunk 8 above)
Comment (PaulWessel) [reply doc/rst/source/psevents.rst]: See above.

File: doc/rst/source/events.rst (hunk 1 above)
Comment (PaulWessel) [reply doc/rst/source/events.rst]: It is trying to show that **+a**, **+f,** **+j** are all independent modifiers (so you can use 0-3 of those) while you can only use either **+r** or **+z** but not both.  Both of those two take an optional argument (_first_ and _format_).  I think it is correctly typeset.

Comment (seisman) [reply doc/rst/source/events.rst]: OK.

**Comment by PaulWessel** [comment]
Getting this:

/Users/pwessel/UH/RESEARCH/CVSPROJECTS/GMTdev/gmt-dev/dbuild/doc/rst/source/supplements/geodesy/earthtide.rst:92: WARNING: Unknown target name: "generate 1d array".

It had a typo before and the ref is correct I think?

File: doc/rst/source/supplements/geodesy/earthtide.rst [hunk 11]
```diff
@@ -95,6 +95,7 @@ Optional Arguments
     current time in UTC from the computer clock. If no **-G** or **-S** are provided then **-T** is interpreted to mean compute
     a time-series at the location specified by **-L**, thus then **-L** becomes mandatory.
     When **-G** and **-T**, only first time T series is considered. Finally, dates may range from 1901 through 2099.
+    For details on array creation, see `Generate 1D Array`_.
```
Comment (seisman) [inline doc/rst/source/supplements/geodesy/earthtide.rst]: You need to add `.. include:: /explain_array.rst_`

Comment (PaulWessel) [reply doc/rst/source/supplements/geodesy/earthtide.rst]: Thanks that worked.

**Pull request merged**